--- a/server/CMakeLists.txt
+++ b/server/CMakeLists.txt
@@ -1,5 +1,6 @@
 
-find_program( MYSQLD_EXECUTABLE mysqld /usr/sbin /usr/local/sbin /usr/libexec /usr/local/libexec /opt/mysql/libexec )
+# MySQLd can be found here on Debian systems
+set( MYSQLD_EXECUTABLE "/usr/sbin/mysqld" CACHE STRING "MySQL Server path" )
 if( MYSQLD_EXECUTABLE)
    message( STATUS "MySQL Server found." )
 else ( MYSQLD_EXECUTABLE )
