Last-Update: 2015-07-14
Forwarded: no
Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=791807
Author: Dmitry Smirnov <onlyjob@member.fsf.org>
Description: initialise PSQL database with "--data-checksums".

--- a/server/src/storage/dbconfigpostgresql.cpp
+++ b/server/src/storage/dbconfigpostgresql.cpp
@@ -219,8 +219,9 @@
     const QString command = QString::fromLatin1( "%1" ).arg( mInitDbPath );
     QStringList arguments;
     arguments << QString::fromLatin1( "--pgdata=%2" ).arg( mPgData )
               // TODO check locale
+              << QString::fromLatin1( "--data-checksums" )
               << QString::fromLatin1( "--locale=en_US.UTF-8" );
     QProcess::execute( command, arguments );
   }
 
