From: Modestas Vainius <modax@debian.org>
Subject: make X11 optional in FindQt4
Forwarded: no

--- a/cmake/modules/FindQt4.cmake
+++ b/cmake/modules/FindQt4.cmake
@@ -1342,7 +1342,7 @@ IF (QT4_QMAKE_FOUND)
   IF(UNIX)
     # on OS X X11 may not be required
     IF (Q_WS_X11)
-      FIND_PACKAGE(X11 REQUIRED)
+      FIND_PACKAGE(X11)
     ENDIF (Q_WS_X11)
     FIND_PACKAGE(Threads)
     SET(QT_QTCORE_LIBRARY ${QT_QTCORE_LIBRARY} ${CMAKE_THREAD_LIBS_INIT})
